### PR TITLE
Feature/timestamp

### PR DESCRIPTION
**Pull request by ssung1**
### What was the feature/problem?

Search wishes by date range.

### How does this PR address the above feature/problem?

Added query parameters to allow user to specify date range.  Or if date range is not specified, a default will be used.

### Check lists (check `x` in `[ ]` of list items)

- [ x] Test passed  (quarantined a few; will fix as soon as we can)
- [ x] Coding style (indentation, etc)

### Additional Comments (if any)

{Please write here}

### CODE REVIEW

File: api/src/routes/__tests__/wish.test.js [hunk 1]
```diff
@@ -15,36 +16,53 @@ describe('Wish route', () => {
...
+    createdAt: '2017-08-14T18:08:56.374Z',
+    updatedAt: '2018-08-14T18:08:56.374Z',    
   }
 
   beforeEach(async () => {
     await Wish.deleteMany({})
   })
 
-  test('It should respond with an array of wishes', async () => {
+  xtest('It should respond with an array of wishes', async () => {
```
Comment (connorjones13) [inline api/src/routes/__tests__/wish.test.js]: did you mean to leave the `x` on `xtest`?

**Comment by liding987** [comment]
@ssung1 there are conflicts needed to be resolved before the merge.

File: api/src/routes/__tests__/wish.test.js (hunk 1 above)
Comment (liding987) [reply api/src/routes/__tests__/wish.test.js]: x means skip I think

Comment (ssung1) [reply api/src/routes/__tests__/wish.test.js]: not really.  i had to disable that test until the update-resource pull request got merged.

I'll update this branch

**Pull request merged**